cdi-jdg/pom.xml

```diff
@@ -86,7 +86,13 @@
         <!-- Infinispan dependency -->
         <dependency>
             <groupId>org.infinispan</groupId>
-            <artifactId>infinispan-embedded</artifactId>
+            <artifactId>infinispan-core</artifactId>
+            <version>${version.org.infinispan}</version>
```

Comment:
Version should not be necessary as we import the bom

Reply:
Yeah... I think something might be wrong in my local product maven repo since it was somehow bringing in version 7.2.3 :\. Let me wipe it out and try again